DEVELOPMENT.md

```diff
@@ -126,7 +126,7 @@ Search the code for org.slf4j.LoggerFactory to see a sample of its use.
 
 ### Use of Java 8 features
 Edgent primary development environment is Java 8, to take advantage of lambda expressions
```

Comment:
nit: Didn't notice this the first time, but there was a typo originally (missing apostrophe). _Edgent_ should be _Edgent's_.